OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -59,7 +59,12 @@ void StartSavingReplay(byte[] initialContent)
 				{
 					file = File.Create(fullFilename);
 				}
-				catch (IOException) { }
+				catch (IOException ex)
+				{
+					if (id > 128)
```

Comment:
Can you please add a comment explaining the significance of 128? We know right now that this is arbitrary, but looking back at this in 5 years might cause confusion over whether 128 is meaningfully different than 100 or 500.

Reply:
Better to have it as a constant with a good name?

Reply:
Sure, but that should still have a comment explaining the motivation for its specific value (unless we want to call it `const int ArbitraryRetryCount` 😁 - but please don't call it that)